src/databricks/labs/remorph/mixins/root_table.py

```diff
@@ -0,0 +1,143 @@
+import logging
```

Comment:
please move to a package named `intermediate`

Reply:
Done